pkg/plugins/backendplugin/instrumentation/instrumentation.go

```diff
@@ -36,6 +36,8 @@ func instrumentPluginRequest(ctx context.Context, cfg *config.Cfg, pluginCtx *ba
 
 	start := time.Now()
 
+	timeBeforePluginRequest := log.TimeSinceStart(ctx, time.Now())
```

Comment:
Nit: Could just use `start` here instead of `time.Now()`

Reply:
:+1: